lib/edit/k_info.txt

```diff
@@ -2095,22 +2087,6 @@ W:5:0:5:0
 A:5/1
 D:This nasty scroll will make a loud noise, waking up foes in your vicinity.
 
-N:203:Trap Creation
-G:?:d
-I:70:7:0
-W:10:0:5:0
-A:10/1
-D:If you read this rather annoying scroll, snares and pitfalls will magically be planted all around
-D:you, ready to do nasty things to you once you walk onto them.
-
-N:204:Trap/Door Destruction
```

Comment:
Bit late for that. Maybe it okay to tunnel through doors instead? Vanilla does that.